wsgi/templates/html/about.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html' %}
```

Comment:
Have I not made it clear earlier that what are the changes expected in this PR? Please cooperate.